packages/web-components/fast-foundation/src/badge/badge.template.ts

```diff
@@ -11,8 +11,8 @@ export const badgeTemplate: FoundationElementTemplate<ViewTemplate<Badge>> = (
     context,
     definition
 ) => html`
-    <template class="${x => (x.circular ? "circular" : "")}">
-        <div class="control" part="control" style="${x => x.generateBadgeStyle()}">
+    <template>
```

Comment:
the `<template>` element should be removed completely in this case since there are no longer any attributes or bindings on it.